Update NFT-RFC-002-use-cases.md (#9)
interNFT/use-cases: NFT-RFC-002-use-cases.md

### 4.1 Quality Carbon Credits

#### Context

> As pressure mounts to balance the yearly global carbon budget due to increasing risk to life, property and markets due to climate related 
> 
> Carbon accounting, markets and politics.

Reviewer: ```suggestion
As pressure mounts to balance the yearly global carbon budget due to increasing risk to life, property and markets due to climate related Carbon accounting, markets and politics.
```

Long lobbied by scientists and citizen groups, corporations and governments are finally taking climate related risk seriously.  As voluntary and compliance markets for carbon, corporate science based reporting, emissions taxes and other mechanisms become more common, the demands for carbon accounting integrity become more challenging to meet.  

> Carbon accounting and science:
> Carbon accounting should follow the natural carbon cycle. Emissions (respiration) and sequestration (inspiration). The global carbon budget must not simply be brought to net zero as quickly as possible, it must also achieve a carbon negative balance for some time to bring us back to pre-industrial levels if we want to guarantee conditions conducive to biosphere functioning as humans have evolved.  

Reviewer: ```suggestion
#### Carbon accounting and science
Carbon accounting should follow the natural carbon cycle: emissions (respiration) and sequestration (inspiration). The global carbon budget must not simply be brought to net zero as quickly as possible, it must also achieve a carbon negative balance for some time to bring us back to pre-industrial levels if we want to guarantee conditions conducive to biosphere functioning as humans have evolved.  
```

> Carbon accounting and markets.
> There are three major types of carbon credits.  
> Avoided emissions (a factory spent money to increase efficiency and can claim credits as they reduce emissions).
> Renewable Energy Credits (a city shifts from fossil fuel based energy to renewable energy and claims credits associated with the quantification of carbon emissions reduction)
> Natural carbon or “living carbon” which is incapsulated by “terraculture” and “mariculture” (AFOLU: Agriculture, Forestry and other Land Use) and Blue Carbon in which sequestration and/or maintenance of carbon stocks in living systems is monitored and quantified.

Reviewer: ```suggestion
#### Carbon accounting and markets
There are three major types of carbon credits:
+ Avoided emissions (a factory spent money to increase efficiency and can claim credits as they reduce emissions).
+ Renewable Energy Credits (a city shifts from fossil fuel based energy to renewable energy and claims credits associated with the quantification of carbon emissions reduction).
+ Natural carbon or “living carbon” which is incapsulated by “terraculture” and “mariculture” (AFOLU: Agriculture, Forestry and other Land Use) and Blue Carbon in which sequestration and/or maintenance of carbon stocks in living systems is monitored and quantified.
```

> Living Carbon User Story:
> 
> User Story One: Interchain credit transfer

Reviewer: ```suggestion
## Living Carbon User Story

### User Story One: Interchain credit transfer
```

Acme Ag and their ecosystem of farmers, processors, and logistics providers, would like to meet the ambitious goals of carbon neutrality by 2030, and even explore if providing carbon credits to industries that cannot hope to achieve carbon negative operations (such as the energy sector), could become a long term profit center.

> Acme ag wants to be able to have accurate carbon reporting internally, as well as participate in markets to buy and sell carbon credits to ensure year over year they can meet carbon accounting goals 1st and position themselves to be a provider of carbon to the market they believe is coming, in partnership with their farmers.

Reviewer: Acme ag -> Acme AG
Acme Ag -> Acme AG

Farmer Jane provides produce to Acme ag.  She would like to add carbon credit sales to her list of produce.  In addition to Acme there are also several other marketplaces forming and Jane wants to be able to find the best possible price for her carbon with the least amount of work.

> A start up carbon market (Market1) approaches her with an easy monitoring and verification onboarding scheme and upfront payment in exchange for her carbon.  She accepts because she needs the cash.  Market1 is running as a smart contract in an Ethereum state channel.  

Reviewer: ```suggestion
A start up carbon market (Market1) approaches her with an easy monitoring and verification onboarding scheme and upfront payment in exchange for her carbon.  She accepts because she needs the cash.  Market1 is running as a smart contract on Ethereum.  
```

Reviewer: "state channel" is not relevant here, and it's also wrongly used. State channel is a link only between 2 entities. Here, a side chain / layer-2 would make more sense.

Acme ag partners with a second marketplace (Market2) which has a higher price, and a different and more rigorous verification threshold. Jane second guesses her decision and registers on Market2 as well as Market1.

> The first round of carbon credits have already been issued. The transition of carbon from Market1 to Market2 requires running additional data collection and uses a different quantification algorithm that is considered by many to be more rigorous and has less uncertainty.  Market2 is running on a cosmosSDK chain.  

Reviewer: ```suggestion
The first round of carbon credits have already been issued. The transition of carbon from Market1 to Market2 requires running additional data collection and uses a different quantification algorithm that is considered by many to be more rigorous and has less uncertainty.  Market2 is running on a Cosmos SDK chain.  
```

Acme ag agrees to underwrite the cost of additional verification.   

Acme ag does not want Farmer Jane's geolocation to be public and desires that this data be kept private and only available to approved monitors and verifiers in the system because they are worried about a competing company poaching their high preforming farmers

#### User Needs
Acme Ag demands that there be no counter carbon claims on the farm in question
Acme Ag demands ability to audit and hold seller liable for double counting
Farmer Jane requires access to multiple markets
Farmer Jane requires monitoring and verification costs to be shouldered by purchaser

> Market1: Need to be paid back for the carbon they prepurchased
> Market2 Needs to be able to burn or represent the movement of that carbon to market2 Needs to be able to represent the provenance of the carbon in question (both geolocation and the fact it moves from a different registry)
> Market2 Needs to be able to update the claim with the new methodology but maintain consistency in the time period and geolocation of the claim

Reviewer: ```suggestion
Market1 needs to be paid back for the carbon they prepurchased.
Market1 needs to be able to burn or represent the movement of that carbon to market2.
Market2 needs to be able to represent the provenance of the carbon in question (both geolocation and the fact it moves from a different registry).
Market2 needs to be able to update the claim with the new methodology but maintain consistency in the time period and geolocation of the claim.
```
